lib/fathead/angularjs/trigger_words.txt

```diff
@@ -0,0 +1,3 @@
+angular js
+angular javascript
+
```

Comment:
Hey, it already triggers for `angularjs` why do we need to add it in `trigger_words.txt` if it automatically triggers. I had added `angularjs` previously but @pjhampton suggested to remove it since it triggers automatically for that. I want final confirmation on your part on this and I would also like to know why it should be added 

![screenshot from 2017-02-25 02-44-53](https://cloud.githubusercontent.com/assets/25108385/23321295/dc96beae-fb04-11e6-8e2e-b47759b930f6.png)